test

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#
+# test - a pa test script
+
+export pa="$(pwd)/pa"
+
+export PA_DIR=test-stuff/passwords
+
+# clean up previous run
+# (the previous state is left around
+# intentionally, in case the dev
+# wants to poke around)
+# that's why we don't clean up on exit
+rm -rf test-stuff
+
+# pa welcomes you
+$pa | grep -q "a simple password manager" ||
+    printf "pa should print a welcome message\n"
+
+# generate pa dirs/identityfile/recipientfile
+$pa list
+
+# pa auto-generated files are correct
+test -s test-stuff/identities ||
+    printf "an identities file should exist\n"
+test -s test-stuff/recipients ||
+    printf "a recipients file should exist\n"
+test -d "$PA_DIR/.git" ||
+    printf "git dir should exist\n"
+# TODO: ensure git author/email are set correctly, etc
+
+# pa add
+printf 'y\n' | $pa add test 2>&1 >/dev/null ||
```

Comment:
\n is not necessary, thanks to `-icanon`

Reply:
fixed in df212eb

Reply:
fixed in [df212eb](https://github.com/biox/pa/pull/48/commits/df212eb779a0bf031940883fa5cb8f4a254798df)